gfx/portraits/portrait_modifiers/99_special.txt

```diff
@@ -261,6 +261,25 @@ long_beard = {
 	}
 }
 
+no_jaw = {
```

Comment:
Not marked with `#Warcraft`

Reply:
Also no trigger on this weight, won't everyone be jaw-less?

Reply:
This only applies if the accessory is marked with the "no_jaw" tag

Reply:
Alright 👍